docs/docs/js/docs.js

```diff
@@ -2,85 +2,92 @@
 hljs.configure({tabReplace: '    '});
 hljs.initHighlighting();
 
-const tocCopy = document.createElement('div');
-tocCopy.id = 'toc-copy';
+function setup() {
```

Comment:
Anything in `docs/` will only ever run in a browser. You made the blanket assumption that all `.js` files are part of the Leaflet code, but some are part of its website. These changes are therefore too broad.